drivers/overlay/overlay.go

```diff
@@ -2036,8 +2053,18 @@ func (g *overlayFileGetter) Get(path string) (io.ReadCloser, error) {
 	return nil, fmt.Errorf("%s: %w", path, os.ErrNotExist)
 }
 
-func (g *overlayFileGetter) Close() error {
-	return nil
+func (g *overlayFileGetter) Close() (Err error) {
+	for _, f := range g.composefsMounts {
+		err := f.Close()
+		if Err == nil {
+			Err = err
+		}
+		err = unix.Rmdir(f.Name())
```

Comment:
Should we logrus.Error(err) on else path, IE If Err != nil, then we should output all or errors, or just wrap all errors?

Reply:
yes that is better.  I've changed it to use `go-multierror` now